packages/server/src/setup/traefik-setup.ts

```diff
@@ -124,6 +124,26 @@ export const initializeTraefik = async ({
 			console.log("No existing container to remove");
 		}
 
+		try {
+			await docker.getImage(imageName).inspect();
+			console.log(`Image ${imageName} already exists locally.`);
+		} catch (error: any) {
+			if (error?.statusCode === 404) {
+				console.log(`Image ${imageName} not found, pulling...`);
+				const stream = await docker.pull(imageName);
+				await new Promise((resolve, reject) => {
+					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+					// @ts-ignore
+					docker.modem.followProgress(stream, (err: Error, res: any) =>
+						err ? reject(err) : resolve(res),
+					);
+				});
```

Comment:
```suggestion
				await new Promise((resolve, reject) => {
					docker.modem.followProgress(stream, (err: Error | null, res: any) =>
						err ? reject(err) : resolve(res),
					);
				});
```

This code pathed the eslint check.
Can you build with it without any errors?